src/components/global-styles.tsx

```diff
@@ -153,6 +153,14 @@ const SizeStyles = css`
     padding: 5px 9px;
     z-index: 1;
   }
+
+  .buy-box {
+    position: relative;
+    background-color: #f5f5f7;
+    border-radius: 12px;
+    outline: 0;
+    transition-property: transform, visibility, opacity;
+  }
```

Comment:
We do we need to add global styles for this feature? Can't we apply these with a component?